Create Shooting

JammieDodgers opened the pull request:
So, for this to work, you'll need to create a prefab to instantiate as a projectile. I've called mine projectile, but the beauty of programming is you can name it "ceiling cat is watching you masturbate", if you want to really confuse hackers. You will also need to designate the first-person camera as main.

JammieDodgers committed:
Create Shooting

So, for this to work, you'll need to create a prefab to instantiate as a projectile. I've called mine projectile, but the beauty of programming is you can name it "ceiling cat is watching you masturbate", if you want to really confuse hackers. You will also need to designate the first-person camera as main.

ghost commented:
I liked that you actually made it to work. May I suggest some changes to optimize your code?
Remove the following line from inside the update
`Rigidbody rb = projectile.GetComponent<Rigidbody> ();`
put Rigidbody rb under GameObject prefab like this
`GameObject prefab;`
`Rigidbody rb`
and finally assign the Rigidbody component from projectile to rb
`rb = projectile.GetComponent<Rigidbody> ();`

Tayloj36 committed:
States for a Turn Based Battle system

Creating a turn based battle system from scratch is far beyond the scope
of the usual github projects, however I've created a starting point for
anyone wishing to do so. This lets you manage the states of combat.

JammieDodgers committed:
Revert "States for a Turn Based Battle system"

This reverts commit 04194604e215fc0a846fb1580acd4b7c6c0370e6.
